Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-4.2.xml

```diff
@@ -0,0 +1,541 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ns4:modelInfo name="QDM" url="urn:healthit-gov:qdm:v4_2" targetQualifier="qdm" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ns4="urn:hl7-org:elm-modelinfo:r1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               patientClassName="QDM.Patient" patientClassIdentifier="Patient" patientBirthDatePropertyName="birth datetime" >
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.Patient" baseType="System.Any" identifier="Patient">
+        <ns4:element name="birth datetime" type="System.DateTime"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.PhysicalExamOrder" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+        <ns4:element name="anatomicalLocationSite" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.ProcedureIntolerance" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="reaction" type="System.Concept"/>
+        <ns4:element name="ordinality" type="System.Concept"/>
+        <ns4:element name="ProcedurePerformed" type="System.Concept"/>
```

Comment:
The `ProcedurePerformed` property looks out of place -- it doesn't reflect a property in QDM 4.2 and it is PascalCased (instead of camelCased).  Was this an intentional addition?  I think that the procedure to which the patient is intolerant would usually be represented by the default `code` property (part of `QDM.QDMBaseType`).  For example, CMS 147v5 uses "Occurrence A of Procedure, Intolerance: Influenza Vaccination".